package.json

```diff
@@ -69,6 +69,7 @@
   },
   "devDependencies": {
     "@types/leaflet": "^1.5.19",
+    "@types/react-router-dom": "^5.3.0",
```

Comment:
why do we need types for react-router-dom?

Reply:
`npm run start` stopped working with the problem that `react-router-dom` is not available, so every place with `NavLink` prevent me to build the application.
I found this solution somewhere on stack overflow.